glslc/src/main.cc

```diff
@@ -97,11 +98,18 @@ bool GetOptionArgument(int argc, char** argv, int* index,
 }  // anonymous namespace
 
 int main(int argc, char** argv) {
-  std::vector<std::pair<std::string, EShLanguage>> input_files;
-  EShLanguage force_shader_stage = EShLangCount;
+  std::vector<std::pair<std::string, shaderc_shader_kind>> input_files;
+  shaderc_shader_kind current_shader_stage = shaderc_glsl_infer_from_source;
```

Comment:
Nit: consider `current_fshader_stage` (note the "f"), because this keeps that flag's value.